src/CMakeLists.txt

```diff
@@ -29,6 +30,7 @@ link_directories (
     ${AO_LIBRARY_DIRS}
     ${RTL_SDR_LIBRARY_DIRS}
     ${FFTW3F_LIBRARY_DIRS}
+    ${LIBUSB_LIBRARY_DIRS}
```

Comment:
Should there be a corresponding entry in `include_directories` above?

Reply:
The include directory isn't really necessary because we don't use libusb ourselves. If we build librtlsdr then we need to be able to link to libusb.